From c5e8c8163c5063d7cc0a376022380f46a9d18ab0 Mon Sep 17 00:00:00 2001
From: Yu Watanabe <watanabe.yu+github@gmail.com>
Date: Wed, 20 Dec 2023 15:08:49 +0900
Subject: [PATCH] man: environment value -> udev property

These are not environment variables, but udev properties.

Follow-up for 3b2e7dc5a285edbbb1bf6aed2d88b889d801613f.

(cherry picked from commit 044149e6152db7a8bb293aac19e84b3b06566d63)

Resolves: RHEL-1317
---
 man/systemd.net-naming-scheme.xml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/man/systemd.net-naming-scheme.xml b/man/systemd.net-naming-scheme.xml
index ec9f3da437..639c03262f 100644
--- a/man/systemd.net-naming-scheme.xml
+++ b/man/systemd.net-naming-scheme.xml
@@ -601,7 +601,7 @@
       <varlistentry>
         <term><varname>ID_NET_NAME_ALLOW=<replaceable>BOOL</replaceable></varname></term>
 
-        <listitem><para>This evironment value sets a fallback policy for reading a sysfs attribute.
+        <listitem><para>This udev property sets a fallback policy for reading a sysfs attribute.
         If set to <constant>0</constant> udev will not read any sysfs attribute by default, unless it is
         explicitly allowlisted, see below. If set to <constant>1</constant> udev can use any sysfs attribute
         unless it is explicitly forbidden. The default value is <constant>1</constant>.</para>
@@ -612,7 +612,7 @@
       <varlistentry>
         <term><varname>ID_NET_NAME_ALLOW_<replaceable>sysfsattr</replaceable>=<replaceable>BOOL</replaceable></varname></term>
 
-        <listitem><para>This evironment value explicitly states if udev shall use the specified
+        <listitem><para>This udev property explicitly states if udev shall use the specified
         <replaceable>sysfsattr</replaceable>, when composing the device name.</para>
 
         </listitem>
